packages/contact-center/station-login/src/station-login/index.tsx

```diff
@@ -19,7 +19,12 @@ const StationLogin: React.FunctionComponent<StationLoginProps> = observer(({onLo
   return <StationLoginPresentational {...props} />;
 });
 
-const WebStationLogin = r2wc(StationLogin);
+const WebStationLogin = r2wc(StationLogin, {
+  props: {
+    onLogin: "function",
+    onLogout: "function"
+  }
+});
```

Comment:
Was this not caught because we didn't build web-components-samples and test in the web-component-samples? This is a breaking change because react and web-component under the same package have different behaviour. 

To catch these types of errors should we have a `Build Samples` step in the pipeline? Since onLogin and onLogout are mandatory arguments, the build should have failed. Il check and update it here.

Reply:
We should also see how we can test this in unit testing. Maybe an argument check AFTER converting the component into a web component

Reply:
This was not caught because the Web Component samples weren't tested.

This should be the ownership of both reviewers and authors to ensure that WC Samples are tested. The way to catch this would be to introduce UI testing like Cypress

@Shreyas281299